mysys/crc32ieee.cc

```diff
@@ -50,9 +50,6 @@ static my_crc32_t init_crc32()
 
 static const my_crc32_t my_checksum_func= init_crc32();
 
-#ifdef __powerpc64__
```

Comment:
The `__powerpc64__` define is true regardless of processor flavor, so this has to go.

Reply:
Right, this is a redundant compile-time guard against multiple definition of `my_checksum()` in case an error in the CMake logic would cause both implementations of  `my_checksum()`.  A duplicated definition should result in a link-time error anyway.